source/best-practices/backup.bat

```diff
@@ -1,37 +1,32 @@
 REM
-REM  目的： 使用 robocopy 命令备份指定目录
-REM  作者： 李天觉、赵志远
-REM  最近修改日期： 2021年2月4日
+REM  Ŀ�ģ� ʹ�� robocopy �����ָ��Ŀ¼
+REM  ���ߣ� ��־Զ�������
+REM  ����޸����ڣ� 2021��2��7��
 REM
-REM  免责声明：
-REM    本脚本完全基于编写者在自己的计算机上备份文件的经验总结，
-REM    并不具备普遍适用性，仅供读者参考。
+REM  ��������
+REM    ���ű���ȫ���ڱ�д�����Լ��ļ�����ϱ����ļ��ľ����ܽᣬ
+REM    �����߱��ձ������ԣ�����߲ο���
 REM
 
-REM 源目录和备份目录
-REM   D:\directory1 和 D:\directory2  : 要备份的源目录
-REM   F:\backup                       : 备份目录
-REM   F:\backup\backup.txt            : 备份目录下的备份日志
+REM ԴĿ¼�ͱ���Ŀ¼
+REM   D:\directory1 �� D:\directory2  : Ҫ���ݵ�ԴĿ¼
+REM   F:\backup                       : ����Ŀ¼
 
-REM robocopy 命令常用选项：
-REM   /mir        : 备份目录成为镜像 （复制子目录；删除备份目录下源目录中不存在的文件和目录）
-REM   /mt[:n]     : 使用 n 个线程进行多线程复制（默认值为 8）。n 至少为 1，但不得大于 128。
-REM                 例如，/mt 表示 8 个线程，/mt:4 表示 4 个线程
-REM   /log+:file  : 将备份状态以追加的方式输出到日志文件 file 中
-REM 可以参考 https://docs.microsoft.com/zh-cn/windows-server/administration/windows-commands/robocopy
-REM 了解 robocopy 命令的更多用法
+REM robocopy �����ѡ�
+REM   /mir        : ����Ŀ¼��Ϊ���� ��������Ŀ¼��ɾ���Ŀ¼��ԴĿ¼�в����ڵ��ļ���Ŀ¼��
+REM   /mt[:n]     : ʹ�� n ���߳̽��ж��̸߳��ƣ�Ĭ��ֵΪ 8����n ����Ϊ 1�������ô��� 128��
+REM                 ���磬/mt ��ʾ 8 ���̣߳�/mt:4 ��ʾ 4 ���߳�
+REM ���Բο� https://docs.microsoft.com/zh-cn/windows-server/administration/windows-commands/robocopy
+REM �˽� robocopy ����ĸ���÷�
 
-REM 注意事项：
-REM   1. 如果命令中有中文，脚本需要是 ANSI 编码格式（Windows 下的默认编码格式）。
-REM      若不确定脚本编码格式，可以选择文件另存为查看，并保存为 ANSI 编码。
-REM   2. 目录名可以不加引号（如 D:\directory1）。但若路径中存在空格，则必须加引号，
-REM      并在目录名最后加空格 "D:\direc tory1 "）。
-REM   3. 使用 /log+:file 选项时，file（如 F:\backup\backup.txt）中的目录（即 F:\backup）必须存在
+REM ע�����
+REM   1. ��������������ģ��ű���Ҫ���� ANSI �����ʽ��Windows �µ�Ĭ�ϱ����ʽ����

```

Comment:
这句话有些问题。
如果是右键新建文本文件，那么默认是 UTF-8；
如果是 CMD 中 `echo 1 > 1.bat` 那么默认也是 UTF-8；
只有在 CMD 中 `echo 测试 > 1.bat` 默认才会是 ANSI。

Reply:
> 如果是右键新建文本文件，那么默认是 UTF-8；
If you write some Chinese, will it become ANSI?
If yes, I think we can ignore this issue since the file with Chinese will be ANSI by default.